Documentation/how-to-build-and-run-ilcompiler-in-console-shell-prompt.md

```diff
@@ -14,13 +14,16 @@ This will result in the following:
 - Build native and managed components of ILCompiler. The final binaries are placed to `<repo_root>\bin\<OS>.<arch>.<Config>\tools`.
 - Build and run tests
 
-# Install .NET Core 2.1 SDK
+## On Windows
 
-* Download .NET Core 2.1 SDK from [https://www.microsoft.com/net/download/core](https://www.microsoft.com/net/download/core)
-* On windows ensure you are using the 'x64 Native Tools Command Prompt for VS 2017'
-    (This is distinct from the 'Developer Command Prompt for VS 2017')
+When use VS 2017:
```

Comment:
Visual Studio 2019 has been around for a while. I would change this to only mention VS 2019. We are likely going drop support for 2017 in not so distant future.